tests/foreman/api/test_permission.py

```diff
@@ -54,12 +54,12 @@ def test_search_by_name(self):
         """
         failures = {}
         for permission_name in self.permission_names:
-            result = entities.Permission(name=permission_name).search()
-            length = len(result)
-            name = result[0]['name'] if length > 0 else None
+            results = entities.Permission(name=permission_name).search()
+            length = len(results)
+            name = results[0].name if length > 0 else None
             if length != 1 or permission_name != name:
                 failures[permission_name] = {
-                    'length': len(result),
+                    'length': len(results),
```

Comment:
this seems awkward to me. can we get rid of the length variable and just use ```len(results)``` in the conditional, or if we keep the variable use it on this line?